packages/space-opera/src/components/model_viewer_snippet/components/open_button.ts

```diff
@@ -244,14 +242,12 @@ export class ImportCard extends LitElement {
         this.selectedDefaultOption = 0;
         return;
       } else if (value in simpleMap) {
```

Comment:
Your maps are key:value pairs, but your keys are called values and I'm not sure what the values (numbers) are used for. Any chance this can be simplified? I haven't read it thoroughly enough to be sure, so feel free to correct me.

Reply:
the dropdown needs an index of what is selected, this was the most straightforward way of doing it. I can look into other options moving forward. I'll change the "value" variable to "key", you're right, that is confusing.